.github/ISSUE_TEMPLATE/Mobile_apps_bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report for Mobile Apps
+about: Create a report to help us improve the Gutenberg mobile apps version
+labels: Mobile App Android/iOS
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Editor version (please complete the following information):**
+- WordPress version: [e.g: 5.3.2]
+- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"]
+- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6, Pixel 3]
+ - OS: [e.g. iOS 8.1, Android 18.1]
+ - WordPress App Version [e.g. 15.3]

```

Comment:
What do you think about adding ", or the git commit hash if it's the demo app" to cover the case the user found the issue on the demo app?